batch/batch/driver/main.py

```diff
@@ -1324,6 +1337,17 @@ async def on_startup(app):
     app['batch_headers'] = {'Authorization': f'Bearer {row["internal_token"]}'}
     app['frozen'] = row['frozen']
 
+    resource_ids = {
+        record['resource']: record['resource_id']
+        async for record in db.select_and_fetchall(
+            '''
+SELECT resource, resource_id FROM resources;
+'''
+        )
+    }
+
+    app['resource_name_to_id'] = resource_ids
```

Comment:
Nit: Could you call `refresh_globals_from_db` here instead of repeating the query?